.github/workflows/ci.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
+          ruby-version: '3.2.2'
```

Comment:
but this wasn't necessary before, right? why is it needed now?

Reply:
![image](https://github.com/rootstrap/rails_api_base/assets/6373536/5daecac2-71ad-49de-93ad-1c7f53b616be)

I think it's automatically set with the `.ruby-version`

Reply:
It's not needed, by default, it uses the `. ruby-version `

Reply:
I'll remove it from all places and check which version it's using

Reply:
hadn't refreshed, you are both right